packages/editor/src/store/test/selectors.js

```diff
@@ -20,10 +20,111 @@ import { RawHTML } from '@wordpress/element';
 /**
  * Internal dependencies
  */
-import * as selectors from '../selectors';
+import { serializeBlocks } from '../actions';
+import * as _selectors from '../selectors';
 import { PREFERENCES_DEFAULTS } from '../defaults';
 import { POST_UPDATE_TRANSACTION_ID } from '../constants';
 
+const selectors = { ..._selectors };
+const selectorNames = Object.keys( selectors );
+selectorNames.forEach( ( name ) => {
```

Comment:
Looking at this custom logic which replaces the production code, I'm wondering whether all the tests in the current form are still valuable. I don't think that having 100-ish lines of complex logic is something we want to maintain in the long run.

Have you considered any alternatives? Why did you decide to take this approach?

Reply:
It's the same registry selector mocking that was being done for `isEditedPostAutosaveable`, but in a general way without having to inline it into every test case.